cmd/snap/cmd_snap_op.go

```diff
@@ -118,10 +118,11 @@ func (x *cmdInstall) Execute([]string) error {
 	var err error
 
 	cli := Client()
-	if strings.Contains(x.Positional.Snap, "/") {
-		uuid, err = cli.InstallSnapFile(x.Positional.Snap)
+	name := x.Positional.Snap
+	if strings.Contains(name, "/") || strings.HasSuffix(name, ".snap") || strings.Contains(name, ".snap.") {
```

Comment:
Common ending for repeated browser downloads and the such (foo.snap.1, etc).  Cannot be ambiguous with a real snap name, due to the two dots.